navigation/presentation.html

```diff
@@ -2,43 +2,1222 @@
+                        &lt;/Tab.Navigator&gt;
+                      &lt;/NavigationContainer&gt;
+                    )
+                  }
+                </code>
+              </pre>
+              <img src="assets/images/tab_navigator.gif" alt="tab navigator">
+            </div>
+
+            <aside class="notes">
+              Tab navigation is a navigation that is tabbed at either the bottom or top of a screen and can be used to switch between different screens.
+              <br /><br />
+              The following example demonstrates how to construct a Bottom Tabs navigator.
+              <br /><br />
+              Tab navigation can take in either the screen as a component or a Stack as the component.
+              <br /><br />
+              Rendering a navigator inside another navigator's screen is called nesting navigators
+            </aside>
+          </section>
+          <!-- 8.3 slide -->
+          <section class="coding-and-picture-slide">
+            <h4 class="slide-name">Drawer navigator</h4>
+            <div>
+              <pre data-id="code-animation">
+                <code class="js" data-trim data-line-numbers>
+                  import { createDrawerNavigator } from ’@react-navigation/drawer‘
+
+                  const Drawer = createDrawerNavigator()
+
+                  function App() {
+                    return(
+                      &lt;NavigationContainer&gt;
+                        &lt;Drawer.Navigator&gt;
+                          &lt;Drawer.Screen name=“Home” component={TabNavigator} /&gt;
+                          &lt;Drawer.Screen name=“Contact” component={ContactScreen} /&gt;
+                        &lt;/Drawer.Navigator&gt;
+                      &lt;/NavigationContainer&gt;
+                    )
+                  }
+                </code>
+              </pre>
+              <img src="assets/images/drawer_navigator.gif" alt="drawer navigator">
+            </div>
+
+            <aside class="notes">
+              Drawer navigation is a slide-out and slide-in drawer that contains links to various screens.
+              <br /><br />
+              The Drawer navigation opens when a menu icon is clicked or when a user swipes their finger from the left or right edge of the app.
+              <br /><br />
+              Stack navigators can be nested within these types of navigators to expand on the content within it.
+            </aside>
+          </section>
+          <!-- 8.4 slide -->
+          <section class="centered-text-and-picture-slide">
+            <h4 class="slide-name">Nesting navigators</h4>
+            <div>
+              <div>
+                <p>deeply nested view hierarchy can be the cause memory and performance issues in lower end devices</p>
+                <p>nesting same type of navigators might lead to a confusing UX</p>
+                <p>code becomes difficult to follow when navigating to nested screens</p>
+              </div>
```

Comment:
it's better to use ul and li tags here to decorate the list with bullet points

Reply:
I would prefer to leave it as it is, because I don't really like how the list with bullet points looks next to the picture, where there are also bullet points